src/aind_data_transfer/jobs/basic_job.py

```diff
@@ -242,6 +244,24 @@ def _compile_metadata(
         processing_file_name = temp_dir / processing_metadata.output_filename
         processing_metadata.write_to_json(processing_file_name)
 
+        created = datetime.utcnow()
+        last_modified = datetime.utcnow()
```

Comment:
You can probably just do `last_modified = created` to avoid calling utcnow() twice